esp/src/eclwatch/CurrentUserDetailsWidget.js

```diff
@@ -51,7 +53,9 @@ define([
 
         //  Hitched actions  ---
         _onSave: function (event) {
-            var dialog = dijit.byId("stubUserDialog");
+            var context = this;
+            var dialog = this.params.Widget;
```

Comment:
Passing an entire Widget via params will mean that this page cannot be displayed on its own or in a react component (which may be ok?)

Reply:
I envision us creating a dialog component that we pass in ...props to replace items such as these in the current world.